Ginger/GingerCoreNET/Database/NoSqlBase/GingerCosmos.cs

```diff
@@ -240,7 +240,7 @@ public override void PerformDBAction()
                             return;
                         }
                         List<PatchOperation> lstPatchOperations = new List<PatchOperation>();
-                        foreach (CosmosPatchInputValues cosmosPatch in Act.CosmosPatchInputValues)
+                        foreach (ActInputValue cosmosPatch in Act.CosmosPatchInputValues)
```

Comment:
Rename cosmosPatch to be generic

Reply:
this code is in gingercosmos.cs hence i named it as such, it is not generic code